libs/blockchain-watcher/feature-app/src/lib/helper/stringHelper.ts

```diff
@@ -17,6 +17,18 @@ export class StringHelper {
     return res
   }
 
+  generateRandomEthPrivateKey(): string {
```

Comment:
I think you could use `ethers.Wallet.createRandom()` instead, the documentation is here: https://docs.ethers.io/v5/api/signer/#Wallet